rise-storage.html

```diff
@@ -1330,8 +1330,7 @@
         decodedUrl = url.substring(0, end !== -1 ? end : url.length);
       }
 
-      // Decode again to handle double slash encodings
```

Comment:
@rodrigopavezi @fjvallarino I don't think that getting %20 back instead of a space is a problem. We only use the name for comparison purposes. We're not actually using it to fetch the file.